crates/subspace-farmer/src/legacy_multi_plots_farm.rs

```diff
@@ -93,61 +91,45 @@ impl LegacyMultiPlotsFarm {
-                        let farming_client = farming_client.clone();
-                        let new_plot = new_plot.clone();
-                        let listen_on = listen_on.clone();
-                        let bootstrap_nodes = bootstrap_nodes.clone();
-                        let first_listen_on = Arc::clone(&first_listen_on);
-
-                        SinglePlotFarm::new(SinglePlotFarmOptions {
-                            base_directory,
-                            plot_index,
-                            max_plot_pieces,
-                            max_plot_size,
-                            total_pieces,
-                            farming_client,
-                            new_plot,
-                            listen_on,
-                            bootstrap_nodes,
-                            first_listen_on,
-                            enable_farming,
-                            reward_address,
-                            enable_dsn_sync,
-                        })
+        let single_plot_farms = tokio::task::spawn_blocking(move || {
+            plot_sizes
+                .par_iter()
+                .enumerate()
+                .map(|(plot_index, &max_plot_pieces)| {
+                    let base_directory = base_directory.join(format!("plot{plot_index}"));
+                    let farming_client = farming_client.clone();
+                    let new_plot = new_plot.clone();
+                    let listen_on = listen_on.clone();
+                    let bootstrap_nodes = bootstrap_nodes.clone();
+                    let first_listen_on = Arc::clone(&first_listen_on);
+
+                    SinglePlotFarm::new(SinglePlotFarmOptions {
+                        base_directory,
+                        plot_index,
+                        max_plot_pieces,
+                        max_plot_size,
+                        total_pieces,
+                        farming_client,
+                        new_plot,
+                        listen_on,
+                        bootstrap_nodes,
+                        first_listen_on,
+                        enable_farming,
+                        reward_address,
+                        enable_dsn_sync,
                     })
-                    .collect::<anyhow::Result<Vec<_>>>()?;
-
-                for (farm, node_runner) in single_plot_farm_instantiations {
-                    if node.is_none() {
-                        node = Some(farm.node.clone());
-                    }
-                    networking_node_runners.push(node_runner);
-                    single_plot_farms.push(farm);
-                }
-
-                Ok::<_, anyhow::Error>((single_plot_farms, networking_node_runners, node))
-            })
-            .await
-            .expect("Not supposed to panic, crash if it does")?;
+                })
+                .collect::<anyhow::Result<Vec<_>>>()
+        })
+        .await
+        .expect("Not supposed to panic, crash if it does")?;
```

Comment:
Kinda useless comment. Says nothing :)

Maybe we can try smth like `Never panics, as underlying code never panics`.

Reply:
Message should explain why it is fine to use `.expect()`. The reason here is that we don't expect any code to panic whatsoever, so if something panics, it is a serious bug and we need to crash the whole app because of it. Given the amount of code behind that call it is hard to confirm that it never panics, so I would consider "Never panics, as underlying code never panics" to be questionable.